src/main/java/ryze/Ryze.java

```diff
@@ -0,0 +1,380 @@
+package ryze;
+
+import ryze.exceptions.InvalidNumberArguments;
+import ryze.exceptions.RyzeException;
+import ryze.task.Deadline;
+import ryze.task.Event;
+import ryze.task.Task;
+import ryze.task.Todo;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.nio.file.Files;
+import java.io.File;
+
+/**
+ * Ryze is a simple command-line bot for managing tasks.
+ */
+public class Ryze {
+
+    private static final String DIVIDER = "➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤➤";
+    private static final String EXIT_COMMAND = "bye";
+    private static final String LIST_COMMAND = "list";
+    private static final String TODO_COMMAND = "todo";
+    private static final String DEADLINE_COMMAND = "deadline";
+    private static final String EVENT_COMMAND = "event";
+    private static final String MARK_COMMAND = "mark";
+    private static final String UNMARK_COMMAND = "unmark";
+    private static final String DELETE_COMMAND = "delete";
+    private static final String TODO = "T";
+    private static final String DEADLINE = "D";
+    private static final String EVENT = "E";
+
+
+
+    private static ArrayList<Task> listOfChatHistory = new ArrayList<>();  // Static task list
+
+    public static void main(String[] args) {
+        initialiseData();
+        initialiseMessage();
+
+        Scanner scanner = new Scanner(System.in);
+        String line;
+        int lastSize = listOfChatHistory.size();
+        while (!(line = scanner.nextLine()).equals(EXIT_COMMAND)) {
+            processCommand(line);
+            lastSize = handleSizeChange(lastSize);
+        }
```

Comment:
You can consider extract this part into a function as well to have the same abstraction level.
E.g.
```
readData();
salary = basic * rise + 1000;
tax = (taxable ? salary * 0.07 : 0);
displayResult();
```
versus
```
readData();
processData();
displayResult();

```